### DISCUSSION TITLE
關於url_for

### DISCUSSION THREAD

**Discussion by briankuo123** [1101-GS3073-C/Python-Community-GS3073-C#64] [Q&A] [answered]
我想請問一下我在網路上查到用: redirect(url_for('user', username = 'your_name'))
這樣的寫法可以將username導入到網址裡,但是實際上跑的時候他卻會回報以下錯誤
想請問一下到底問題出在哪? 謝謝各位!
![image](https://user-images.githubusercontent.com/85862988/137627531-a064e79f-7622-4e65-88a6-a6579490a33d.png)
![image](https://user-images.githubusercontent.com/85862988/137627543-22971805-332c-4424-8ebc-3d68f024e9e9.png)
![image](https://user-images.githubusercontent.com/85862988/137627567-12ff09dd-a259-49ce-9df2-feb8583cc596.png)

**Comment by jack90325** [comment, marked as answer]
你試看看redirect(url_for('show_user_profile', username = result[0]))

**Reply by jack90325** [reply]
因為url_for是顯示那個函式的位置

**Reply by bonny123865** [reply]
非常謝謝你，我也順便一起得救了！
原本也在想一樣的問題
